src/rdsn/src/common/backup.thrift

```diff
@@ -64,24 +64,28 @@ struct configuration_restore_request
     9:optional string   restore_path;
 }
 
+// meta -> replica
 struct backup_request
```

Comment:
Will you use another RPC code? Is there any problem if a user use old version shell-tool  attempt to control the cluster?

Reply:
This rpc is not used from shell-tool to meta server, is meta server to replica server, won't trigger the control problem.

Reply:
OK, thanks.
I meant how to keep compatablity, if the meta servers are in new version, and the replica server are in old version, what will happen if we ask the cluster to do backup? Is it neccessary to add new rpc code for the new implemention.

Reply:
If meta server is in new version, replica server in old version. I don't consider this condition compatible, I think it is a dangerous case for meta server and replica server has different version, because new version especially a feature version, meta server will provide mamy new rpc which replica server can not recognize, and it is not necessary to add new rpc code for compatible only when new meta and old replica case.

Reply:
Rolling update is a common case which many user have to face. We will not ensure every feature should work well at this case, but at least avoid the server crash.
If we rewrite the RPC message, the related RPC code would better to add a new one, and left the old one as deprecated.

Reply:
Rolling update is a common case, but it is not recommended to update meta firstly, we recommendly update replica server firstly, so it is a seldom case that meta is new version but replica is old version.

Reply:
No matter which one is new version. The new replica server will still crash if receive a old backup request from old versio meta server?